src/app/events/page.tsx

```diff
@@ -123,8 +129,13 @@ export default function Events({ searchParams }: Props) {
 
       <PageSection kicker="Events" title="Network Events">
         <FilterContainer>
-          <FilterContainer.ResultsAndCategory
-            results={<ResultsAndReset results={categorizedResults.length} />}
+          <FilterContainer.ResultsAndCategoryCompact
+            results={
+              <CountAndReset
+                counts={filteredCategoryCounts}
+                isSelected={isResetCategoryApplied(searchParams)}
```

Comment:
The name of the function `isResetCategoryApplied` is a bit misleading. What about `areFiltersReset`?